.github/workflows/_build_llvm_libraries.yml

```diff
@@ -0,0 +1,54 @@
+# CALLED WORKFLOW
+# 1. You can connect up to four levels of workflows.
+# 2. Reusable workflows stored within a private repository can only be used by workflows within the same repository.
+# 3. Any environment variables set in an env context defined at the workflow level in the caller workflow
+#    are not propagated to the called workflow.
+name: Reusable workflow-build_llvm_libraries&build_lldb_if_needed
+
+on:
+  workflow_call:
+    inputs:
+      runs-on:
+        required: true
+        # can't pass array now, workaround by passing Json string and deserialize later
+        default: "['ubuntu-20.04', 'ubuntu-22.04']"
+        type: string
+      build-lldb:
```

Comment:
there will be a specific workflow for lldb building. So `build-lldb` could be removed.